### PR TITLE
OCPBUGS-18270: Add readme for e810 plugin

### PR DESCRIPTION
**Pull request by josephdrichard**

### CODE REVIEW

File: addons/intel/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    ts2phcOpts: " "
+    ts2phcConf: |
+      [nmea]
+      ts2phc.master 1
+      [global]
+      use_syslog  0
+      verbose 1
+      logging_level 7
+      ts2phc.pulsewidth 100000000
+      #GNSS module s /dev/ttyGNSS* -al use _0
```
Comment (aneeshkp) [inline addons/intel/README.md:97]: we can remove this line

File: addons/intel/README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  namespace: openshift-ptp
+spec:
+  profile:
+  - name: "profile1"
+    ...
+    ...
+    ......   
+    plugins:
+      e810:
+        enableDefaultConfig: false
```
Comment (aneeshkp) [inline addons/intel/README.md:29]: can we make this as by default  false? at this point this is usable only for non WPC or if you want to use ts2phc only , correct ?

**Comment by josephdrichard** [comment]
> do we have plugin setting for localMaxOffset and LocalHoldOver values ?

updated

File: addons/intel/README.md (hunk 2 above)
Comment (josephdrichard) [reply addons/intel/README.md:29]: already by default is false

**Pull request merged**